docker-compose.yml

```diff
@@ -46,11 +46,14 @@ x-op-backend: &backend
     OPENPROJECT_WEB_MAX__THREADS: 1
     OPENPROJECT_WEB_MIN__THREADS: 1
     OPENPROJECT_WEB_WORKERS: 0
+    PIDFILE: /home/dev/openproject/tmpfs/pids/server.pid
```

Comment:
❓ Where is this env variable used?

Reply:
I read the linked blog post, especially [this section](https://ieftimov.com/posts/docker-compose-stray-pids-rails-beyond/#solution-use-tmpfs). It explains, that `PIDFILE` apparently is interpreted by rails itself. -> resolved

Reply:
For reference: 

https://github.com/rails/rails/blob/cacb8475a9d4373c0db437e7be4905685f03cefa/railties/lib/rails/commands/server/server_command.rb#L245-L249